bundler/lib/bundler/friendly_errors.rb

```diff
@@ -78,7 +78,7 @@ def request_issue_report_for(e)
       Bundler.ui.error <<~EOS, nil, :yellow
 
         First, try this link to see if there are any existing issue reports for this error:
-        #{issues_url(e)}
+        #{discussions_url(e)}
 
         If there aren't any reports for this error yet, please fill in the new issue form located at #{new_issue_url}, and copy and paste the report template above in there.
```

Comment:
This changes `new_issue_url` to `new_discussion_url`, to match my other suggestion.

```suggestion
        If there aren't any reports for this error yet, please fill in the new issue form located at #{new_discussion_url}, and copy and paste the report template above in there.
```